File: src/BukkitPE/command/defaults/StopCommand.php
<?php
namespace BukkitPE\command\defaults;

use BukkitPE\command\Command;
use BukkitPE\command\CommandSender;
use BukkitPE\event\TranslationContainer;


class StopCommand extends VanillaCommand{

	public function __construct($name){
		parent::__construct(
			$name,
			"%BukkitPE.command.stop.description",
			"%commands.stop.usage"
		);
		$this->setPermission("BukkitPE.command.stop");
	}

	public function execute(CommandSender $sender, $currentAlias, array $args){
		if(!$this->testPermission($sender)){
			return true;
		}

		Command::broadcastCommandMessage($sender, new TranslationContainer(TextFormat::RED . "commands.stop.start"));
		$sender->getServer()->setshutdownreason(implode(" ", $args));
		$sender->getServer()->shutdown();

		return true;
	}
}

File: src/net/BukkitPE/command/defaults/StopCommand.java
package net.BukkitPE.command.defaults;

import net.BukkitPE.command.Command;
import net.BukkitPE.command.CommandSender;
import net.BukkitPE.event.TranslationContainer;

/**
 * author: MagicDroidX
 * BukkitPE Project
 */
public class StopCommand extends VanillaCommand {

    public StopCommand(String name) {
        super(name, "%BukkitPE.command.stop.description", "%commands.stop.usage");
        this.setPermission("BukkitPE.command.stop");
    }

    @Override
    public boolean execute(CommandSender sender, String commandLabel, String[] args) {
        if (!this.testPermission(sender)) {
            return true;
        }

        Command.broadcastCommandMessage(sender, new TranslationContainer("commands.stop.start"));

        sender.getServer().shutdown();

        return true;
    }
}
